typings/internalBinding/code_integrity.d.ts

```diff
@@ -0,0 +1,6 @@
+export interface CodeIntegrityBinding {
+    isAllowedToExecuteFile(filePath: string) : boolean;

```

Comment:
The indentation is wrong.

```suggestion
  isAllowedToExecuteFile(filePath: string) : boolean;

```